### PR TITLE
[MERL-697] Tests: Add tests for <Toolbar />

### PR DESCRIPTION
**Pull request by theodesp**
## Tasks

- [ ] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.

## Description

Adds unit tests for <Toolbar />

## Testing

Unit tests should all pass.

## Screenshots

Screenshot 2023-03-03 at 15 38 27

### CODE REVIEW

File: packages/faustwp-core/tests/components/Toolbar/Toolbar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            id: 'test-node',
+            location: 'primary',
+            component: <div>Test Node</div>,
+          },
+        ];
+      },
+    );
+  }
+}
+
```
Comment (matthewguywright) [inline packages/faustwp-core/tests/components/Toolbar/Toolbar.test.tsx:161]: Awesome tests!

**Review by matthewguywright — APPROVED** [review]

File: packages/faustwp-core/src/components/Toolbar/Toolbar.tsx [hunk 2]
```diff
@@ -155,7 +155,10 @@ export function Toolbar({ seedNode }: ToolbarProps) {
         className="quicklinks"
         role="navigation"
         aria-label="Toolbar">
-        <ul id="wp-admin-bar-root-default" className="ab-top-menu">
+        <ul
+          id="wp-admin-bar-root-default"
+          className="ab-top-menu"
+          aria-label="primary-toolbar-nodes">
```
Comment (josephfusco) [inline packages/faustwp-core/src/components/Toolbar/Toolbar.tsx:161]:
```suggestion
          aria-label="primary toolbar menu items">
```

Comment (josephfusco) [reply packages/faustwp-core/src/components/Toolbar/Toolbar.tsx:161]: Nice add!
This should use human readable language that a WordPress publisher might understand.

File: packages/faustwp-core/src/components/Toolbar/Toolbar.tsx [hunk 3]
```diff
@@ -165,7 +168,8 @@ export function Toolbar({ seedNode }: ToolbarProps) {
         </ul>
         <ul
           id="wp-admin-bar-top-secondary"
-          className="ab-top-secondary ab-top-menu">
+          className="ab-top-secondary ab-top-menu"
+          aria-label="secondary-toolbar-nodes">
```
Comment (josephfusco) [inline packages/faustwp-core/src/components/Toolbar/Toolbar.tsx:172]:
```suggestion
          aria-label="secondary toolbar menu items">
```

File: packages/faustwp-core/tests/components/Toolbar/Toolbar.test.tsx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+  expect.assertions(1);
+  mockIsAuthenticated = true;
+  const dom = render(<Toolbar />);
+  const navElement = await waitFor(() =>
+    queryByAttribute('id', dom.container, 'wpadminbar'),
+  );
+  expect(navElement).toBeInTheDocument();
+});
+
+test('renders a default list of nodes in the primary section if seedNode is not provided', async () => {
```
Comment (josephfusco) [inline packages/faustwp-core/tests/components/Toolbar/Toolbar.test.tsx:42]: 🔥

**Review by josephfusco — CHANGES_REQUESTED** [review]
Awesome work on covering these scenarios @theodesp!

**Review by josephfusco — APPROVED** [review]
Nice work @theodesp!

**Pull request merged**